pkg/csi/service/common/util.go

```diff
@@ -354,6 +355,7 @@ func ConvertVolumeHealthStatus(volHealthStatus string) (string, error) {
 		// This implies the volume does not exist any more.
 		// Set health annotation to "Inaccessible" so that
 		// the caller can make appropriate reactions based on this status
+		log.Debugf("SPBM returns health status %q", volHealthStatus)
```

Comment:
Actually SPBM did not really return the health status and its empty string. So, this error message will show up as `SPBM returns health status`. Can we fix this message?

Reply:
Fixed. Change the message.